aws/build-task-definition/build-task-definition.sh

```diff
@@ -47,8 +55,8 @@ if [ -n "$CURRENT_STABLE_TASKDEF_ARN" ]; then
   get_task_definition "$(echo "${CURRENT_STABLE_TASKDEF_ARN}" | tr  -d '\n')"
   current_stable_taskdef="$returned_task_definition"
 
-  current_tmp="$(mktemp)"; printf '%s\n' "$current_stable_taskdef" | jq -S . > "$current_tmp"
-  new_tmp="$(mktemp)";     printf '%s\n' "$new_task_definition"    | jq -S . > "$new_tmp"
```

Comment:
Before, we were using `printf` because `echo` was missing something that I can't quite remember 🤔 (maybe the last new line?)

Are you sure an `echo` in `pretty_print_task_definition()` will be enough?

Reply:
I tested and everything worked with `echo`